llvm/test/tools/llvm-objcopy/ELF/srec-writer.test

```diff
@@ -111,36 +116,39 @@ FileHeader:
   Machine:         EM_X86_64
 Sections:
   - Name:            .text1
-# Part of section data is in 32-bit address range
-# and part isn't.
+## Part of section data is in 32-bit address range
+## and part isn't.
     Type:            SHT_PROGBITS
     Flags:           [ SHF_ALLOC]
     Address:         0xFFFFFFF8
     Content:         "000102030405060708"
   - Name:            .text2
-  # Entire secion is outside of 32-bit range.
+  ## Entire secion is outside of 32-bit range.
     Type:            SHT_PROGBITS
     Flags:           [ SHF_ALLOC ]
     Address:         0xFFFFFFFF0
     Content:         "0001020304"
 
-## Records should use segment physical addresses
-# RUN: yaml2obj %s --docnum=3 -o %t.seg
-# RUN: llvm-objcopy -O srec %t.seg - | FileCheck %s --check-prefix=PADDR
+## This tests an input file with segments and expects
+## physical addresses instead of virtual addresses.
 
```

Comment:
Nit: as above, delete this blank line to tie the comment to the associated RUN lines.

Reply:
Done